app/src/main/java/openfoodfacts/github/scrachx/openfood/hilt/ApiModule.kt

```diff
@@ -0,0 +1,33 @@
+package openfoodfacts.github.scrachx.openfood.hilt
+
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import openfoodfacts.github.scrachx.openfood.network.services.AnalysisDataAPI
+import openfoodfacts.github.scrachx.openfood.network.services.ProductsAPI
+import openfoodfacts.github.scrachx.openfood.network.services.RobotoffAPI
+import openfoodfacts.github.scrachx.openfood.network.services.WikidataAPI
+import retrofit2.Retrofit
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object ApiModule {
+
+    @Provides
+    @Singleton
+    fun provideProductsAPI(retrofit: Retrofit): ProductsAPI = retrofit.create(ProductsAPI::class.java)
+
+    @Provides
+    @Singleton
+    fun provideWikiDataAPI(retrofit: Retrofit): WikidataAPI = retrofit.create(WikidataAPI::class.java)
```

Comment:
Wikidata does not use the same base url, and same for robotoff I think

Reply:
oh right, fixed